test/javascriptsource/nativemobileresources/actions/HasNotificationPermission.js

```diff
@@ -19,11 +19,16 @@ import { NativeModules } from "react-native";
 export async function HasNotificationPermission() {
 	// BEGIN USER CODE
     // Documentation https://rnfirebase.io/docs/v5.x.x/notifications/receiving-notifications
-    if (NativeModules && !NativeModules.RNFirebase) {
+    if (NativeModules && !NativeModules.RNFBMessagingModule) {
         return Promise.reject(new Error("Firebase module is not available in your app"));
     }
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const firebase = require("react-native-firebase");
-    return firebase.messaging().hasPermission();
+    return NativeModules.RNFBMessagingModule.hasPermission().then((authStatus) => {
+        if (authStatus) {
+            return Promise.resolve(true);
+        }
+        else {
+            return Promise.resolve(false);
+        }
```

Comment:
This could also be a one liner; `return Promise.resolve(authStatus ? true : false)`

Reply:
This JS action is part of the `Native Mobile Resources` module, which is used in the project but maintained in https://github.com/mendix/widgets-resources